src/main/java/net/dv8tion/jda/api/hooks/ListenerAdapter.java

```diff
@@ -111,12 +112,15 @@ public void onGenericUpdate(@Nonnull UpdateEvent<?, ?> event) {}
     public void onRawGateway(@Nonnull RawGatewayEvent event) {}
     public void onGatewayPing(@Nonnull GatewayPingEvent event) {}
 
-    //JDA Events
+    //Session Events
     public void onReady(@Nonnull ReadyEvent event) {}
-    public void onResumed(@Nonnull ResumedEvent event) {}
-    public void onReconnected(@Nonnull ReconnectedEvent event) {}
-    public void onDisconnect(@Nonnull DisconnectEvent event) {}
+    public void onSessionInvalidate(@Nonnull SessionInvalidateEvent event) {}
+    public void onSessionDisconnect(@Nonnull SessionDisconnectEvent event) {}
+    public void onSessionResumed(@Nonnull SessionResumedEvent event) {}
+    public void onSessionRecreate(@Nonnull SessionRecreateEvent event) {}
```

Comment:
```suggestion
    public void onSessionInvalidate(@Nonnull SessionInvalidateEvent event) {}
    public void onSessionDisconnect(@Nonnull SessionDisconnectEvent event) {}
    public void onSessionResumed(@Nonnull SessionResumedEvent event) {}
    public void onSessionRecreate(@Nonnull SessionRecreateEvent event) {}
```
`Resumed` is the only past-tense version here. We should probably make these all the same tense

Reply:
Which tense would be ideal?

Reply:
"Resumed" could technically be present tense here (specifically present perfect tense) but I see the confusion. Regardless, I still think present tense is fine here, since we do "onGuildLeave" and not "onGuildLeft" for example.